Philips.CodeAnalysis.DuplicateCodeAnalyzer/AvoidDuplicateCodeAnalyzer.cs

```diff
@@ -674,7 +674,7 @@ public virtual Func<LocationEnvelope> MakeFullLocationEnvelope(TokenInfo firstTo
 
 			(var components, var hash) = _hashCalculator.ToComponentHashes();
 
-			Evidence e = new Evidence(MakeFullLocationEnvelope(firstToken, token), components, hash);
+			Evidence e = new(MakeFullLocationEnvelope(firstToken, token), components, hash);

```

Comment:
Not sure if this line got any better to read. Consider a local variable for the result of `MakeFullLocationEnvelope`

Reply:
Fixed, and created Issue to investigate automation.